documentation/src/main/asciidoc/configuration.asciidoc

```diff
@@ -159,4 +159,19 @@ Another interesting property is
 `deploymentExportPath` which is optional and instructs
 Arquillian to dump the test artifacts to the specified directory on disk.
 Inspection of the deployed artifacts can be very useful when debugging
-test failures.
\ No newline at end of file
+test failures.
+
+[[configuration-javafx]]
+
+The Bean Validation specification mandates a support of JavaFX if JavaFX is available in the classpath.
+
+While JavaFX is included in the Oracle JDK and some other JDKs include https://wiki.openjdk.java.net/display/OpenJFX/Main[OpenJFX],
+it might not be included in all JDKs.
+
+Having JavaFX available in a container environment might also not be straightforward.
+
+For these reasons, the JavaFX tests are disabled by default when running the TCK with the default options.
```

Comment:
Redundant "default": ... are disabled when running the TCK with the default options.

Reply:
Fixed.